### PR TITLE
Fix apostrophes

### PR DESCRIPTION
**Pull request by doktorbro**

### CODE REVIEW

File: about.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: About
 layout: default
 permalink: /about/
-title: What's this about?
+title: What’s this about?
```
Comment (haacked) [inline about.md:5]: Interesting. I'm surprised this wasn't converted using smart quotes. Seems like a bug in Jekyll. What do you think @benbalter?

**Comment by haacked** [comment]
Thanks! We love the attention to detail. :thumbsup:

File: about.md (hunk 1 above)
Comment (doktorbro) [inline about.md:5]: No, it’s not a bug. You must explicitly pass the content to a markdown renderer, if you want rendering outside of `{{content}}`. Use the liquid filter `markdownify` here https://github.com/github/choosealicense.com/blob/gh-pages/_includes/header.html#L6

Comment (haacked) [inline about.md:5]: Ah! Mind sending a PR that does that?

**Pull request merged**